src/ringct/multiexp.h

```diff
@@ -55,17 +61,19 @@ struct MultiexpData {
 };
 
 struct straus_cached_data;
-struct pippenger_cached_data;
+using pippenger_cached_data = std::vector<ge_cached, boost::alignment::aligned_allocator<ge_cached, 4096>>;
```

Comment:
The old PR did aligned allocs as well. Do you know why?

Reply:
No clue, I just figured it's best to keep that in, in case it's somehow important.

Reply:
I think @moneromooo-monero added this feature IIRC, so maybe we ask him

Reply:
Looks like it showed up in a commit called 'some speedups': https://github.com/monero-project/monero/pull/4219

Reply:
4096 is the virtual page size on x86, maybe this is why. It will only matter in low memory conditions when swapping starts - then the cache will fit in smaller number of pages. This alignment shouldn't affect performance otherwise.